bucket/processhacker-nightly.json

```diff
@@ -25,6 +25,10 @@
             "Process Hacker"
         ]
     ],
+    "persist": [
+        "ProcessHacker.exe.settings.xml",
```

Comment:
What does this file contain? I don't think it has user data.

Reply:
It contains Process Hacker settings as the filename suggests.